apps/web/src/components/tx/security/blockaid/useBlockaid.ts

```diff
@@ -22,23 +22,29 @@ export const useBlockaid = (
   const { safe, safeAddress } = useSafeInfo()
   const signer = useSigner()
   const isFeatureEnabled = useHasFeature(FEATURES.RISK_MITIGATION)
+  const jsonData = data && JSON.stringify(data)
```

Comment:
Stringified this because it was spamming requests every time data is updated, and data was updating every 15s.